_config.yml

```diff
@@ -0,0 +1,14 @@
+markdown: kramdown
```

Comment:
This is for Github Pages? This should go on the `gh-pages` branch instead of `master`.